src/domain/settlement/index.js

```diff
@@ -163,12 +163,10 @@ module.exports = {
         })
         return result
       } else {
-        const err = new Error('Settlements not found')
-        throw err
+        throw ErrorHandler.Factory.createInternalServerFSPIOPError('Settlements not found')
       }
     } else {
-      const err = new Error('Use at least one parameter: state, fromDateTime, toDateTime, currency, settlementWindowId, fromSettlementWindowDateTime, toSettlementWindowDateTime, participantId, accountId')
-      throw err
+      throw ErrorHandler.Factory.createInternalServerFSPIOPError('Use at least one parameter: state, fromDateTime, toDateTime, currency, settlementWindowId, fromSettlementWindowDateTime, toSettlementWindowDateTime, participantId, accountId')
```

Comment:
Perhaps this could be a 3100 generic validation error or a 3102 Missing mandatory element error?

Reply:
Set to Validation error (3100)